internal/util/update/replace.go

```diff
@@ -26,13 +26,16 @@ import (
 type ReplaceUpdater struct{}
 
 func (u ReplaceUpdater) Update(options UpdateOptions) error {
+	if options.KptFile.UpstreamLock == nil || options.KptFile.UpstreamLock.GitLock == nil {
+		return nil
+	}
 	options.KptFile.UpstreamLock.GitLock.Ref = options.ToRef
 	options.KptFile.UpstreamLock.GitLock.Repo = options.ToRepo
-	return fetch.Command{
-		RepoSpec: &git.RepoSpec{
-			OrgRepo: options.KptFile.UpstreamLock.GitLock.Repo,
-			Ref:     options.KptFile.UpstreamLock.GitLock.Ref,
-			Path:    options.KptFile.UpstreamLock.GitLock.Directory,
+	return get.Command{
```

Comment:
We should use get instead of fetch here as this step should also pull the remote subpackages similar to other update strategies.